doc/source/building/windows.rst

```diff
@@ -168,90 +281,115 @@ the following in your build log:**
 
    FOUND:
       libraries = ['openblas']
-      library_dirs = ['C:\opt\64\lib']
-      language = c
+      library_dirs = ['C:\...........\lib']
+      language = f77
       define_macros = [('HAVE_CBLAS', None)]
 
+We'll get back to this below how to check.
+
 Building SciPy
 ==============
 
-Once you have built OpenBLAS, it's time to build SciPy. Before continuing make sure to
-install the following software for building on the latest Python version. For building
-on other Python versions, see the WindowsCompilers_ page.
+Once you have built OpenBLAS, it's time to build SciPy. Before continuing make
+sure to install the following software for building on the latest Python
```

Comment:
Would you recommend a particular distribution? 
Maybe note which version this was tested with (looks like 3.8 below)?
I use Anaconda, so the Python on my path is:
```
Python 3.7.3 (default, Apr 24 2019, 15:29:51) [MSC v.1915 64 bit (AMD64)] :: Anaconda, Inc. on win32
```
Should the instructions below work if I create a new environment with Python3.8 and activate that environment before proceeding?

Reply:
In Anaconda it should compile without all these shananigans since it ships out with MKL no?

Reply:
Come to think of it, I really don't know anything about the Conda side ☹️

Reply:
Yes it does come with MKL. Do you want me to try skipping the OpenBLAS build and start building SciPy within an Anaconda environment?

Reply:
It won't hurt I guess. However I don't know where things are located on a conda installation so it might not find the MKL library. This conda storyline should be explained separately I think. But fingers crossed...

Reply:
Yeah. Separately is fine for now. At some point it would be nice to figure out how to manage all of these better, especially when we document non-conda builds on Mac and Linux. Many instructions are common, so it would be nice if we had some way of reusing that material.